src/BenchmarksApps/TechEmpower/PlatformBenchmarks/BenchmarkApplication.Fortunes.cs

```diff
@@ -43,6 +45,37 @@ private void OutputFortunes(PipeWriter pipeWriter, List<Fortune> model)
             // Date header
             writer.Write(DateHeader.HeaderBytes);
 
+            var bodyStart = writer.Buffered;
+            // Body
+            writer.Write(_fortunesTableStart);
+            foreach (var item in model)
+            {
+                writer.Write(_fortunesRowStart);
+                writer.WriteNumeric((uint)item.Id);
+                writer.Write(_fortunesColumn);
+                HtmlEncoder.EncodeUtf8(item.Message.Span, writer.Span, out var bytesConsumed, out var bytesWritten, isFinalBlock: true);
+                Debug.Assert(bytesConsumed == item.Message.Length, "Not enough remaining space in the buffer");
+                writer.Advance(bytesWritten);
```

Comment:
Might be more aesthetically pleasing to factor this into an extension/instance method on `writer` like `WriteHtmlEncoded(ReadOnlySpan<byte> content)`

Reply:
Yeah, I thought about it; but we need the HtmlEncoder instance, which we currently cache at the application level; BufferWrite is a ref struct, so we'd have to pass it in each time we create it, which seems a bit much.

In any case, I think the current code looks pretty decent all in all... We can always prettify in the future.

Reply:
Well it will all change anyway when I merge my Razor PR :smile: